### ISSUE TITLE
Add support and ability to mark Quality stat on all items.

### ISSUE THREAD

**Issue by Xeverous** [SnosMe/awakened-poe-trade#1450]
Since 3.25, >20% Quality is now generally drop-only which makes bases with high quality very valuable.

APT recognizes quality on flasks, but not on all equipment. Please make it so that one can always mark Quality stat, just like on flasks

![image](https://github.com/user-attachments/assets/982d1793-c59f-4177-b6bb-23f4e0fe104d)
![image](https://github.com/user-attachments/assets/57b4d557-8ad7-49cb-9f8d-656ad79cd80c)

Item in first screen:

```
Item Class: Body Armours
Rarity: Magic
Superior Sacred Chainmail
--------
Quality: +24% (augmented)
Armour: 957 (augmented)
Energy Shield: 192 (augmented)
--------
Requirements:
Str: 173
Int: 173
--------
Sockets: R-R R 
--------
Item Level: 84
--------
Unidentified
```

**Comment by iParadoxG** [comment]
Also please add support to search only after selecting search while price checking gems. 

For eg. if I have a petrified blood gem with 12% quality, and I price check it, it starts the search ASAP. and when I click on the quality button, it searches Again. The same is true when I click corrupted button. I don't want to make several searched to poe.trade when i PC items. please make it so the search is only made when I click on the search button.

**Comment by Xebtria** [comment]
> Also please add support to search only after selecting search while price checking gems.
> 
> For eg. if I have a petrified blood gem with 12% quality, and I price check it, it starts the search ASAP. and when I click on the quality button, it searches Again. The same is true when I click corrupted button. I don't want to make several searched to poe.trade when i PC items. please make it so the search is only made when I click on the search button.

this should be its own issue / feature request.

**Comment by soderda** [comment]
I went ahead and created a [pull request](https://github.com/SnosMe/awakened-poe-trade/pull/1452) for this feature. It will only work for items with a quality of above 20% though, but that's when you would want it.

**Comment by Xeverous** [comment]
Has this version already been released? I see my own installation is too new:

![image](https://github.com/user-attachments/assets/0e8b04ed-18dc-488c-bd80-9792e5326035)

https://snosme.github.io/awakened-poe-trade/download says that "Latest version is 3.25.101"

**Comment by livejamie** [comment]
@SnosMe Can we get this pull request approved?

**Comment by Max-Burns** [comment]
This would be really nice.

**Comment by livejamie** [comment]
Is this something that can be merged? These are some of the most expensive drops and they have to be manually pricechecked.

Also mentioned in #1460 #1548

**Comment by vincentsels** [comment]
[<img width="478" height="103" alt="Image" src="https://github.com/user-attachments/assets/c37ac9dd-10a0-4eb9-ba67-bf4630c106dd" />](https://www.reddit.com/r/pathofexile/comments/1eqmmwc/psa_awakened_poe_trade_tool_cannot_check_quality/)


I guess he never made it home? :(

**Comment by Xebtria** [comment]
newest version 3.26.102 says it is now implemented

<img width="500" height="167" alt="Image" src="https://github.com/user-attachments/assets/57276ba4-c849-4127-b9f3-ee3f90713a45" />

**Comment by Xeverous** [comment]
great
